code/__DEFINES/is_helpers.dm

```diff
@@ -79,7 +80,7 @@ GLOBAL_LIST_INIT(turfs_openspace, typecacheof(list(
 #define iszombie(A) (is_species(A, /datum/species/zombie))
 #define isskeleton(A) (is_species(A, /datum/species/skeleton))
 #define ismoth(A) (is_species(A, /datum/species/moth))
-#define isfelinid(A) (is_species(A, /datum/species/human/felinid))
+#define isfelinid(A) (is_species(A, /datum/species/humanoid/felinid))
```

Comment:
should still be /human

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/68916/commits/baa892edfd2c7ce3456ec647c5259a59bebd4ef8